tpgtools/overrides/compute/beta/network_firewall_policy_association.yaml

```diff
@@ -0,0 +1,29 @@
+- type: CUSTOM_RESOURCE_NAME
+  details:
+    title: region_network_firewall_policy_association
+  location: region
+- type: EXCLUDE
+  field: location
+  location: global
+- type: EXCLUDE
+  field: region
+- type: CUSTOM_NAME
+  details:
+    name: region
+  field: location
+- type: CUSTOM_ID
+  details:
+    id: projects/{{project}}/regions/{{region}}/firewallPolicies/{{firewall_policy}}/getAssociation?name={{name}}
```

Comment:
For consistency with FirewallPolicy, let's use `projects/{{project}}/regions/{{region}}/firewallPolicies/{{firewall_policy}}/associations/{{name}}` rather than `projects/{{project}}/regions/{{region}}/firewallPolicies/{{firewall_policy}}/getAssociation?name={{name}}` for the import and id formats. See https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_firewall_policy_association#import

Reply:
thanks, fixed in latest commit

Reply:
We'll want to amend the id format as well, on this line!